include/aws/http/connection.h

```diff
@@ -83,13 +83,16 @@ typedef void(aws_http2_on_ping_complete_fn)(
  *      peer, and are safe to retry on another connection.
  * @param http2_error_code The HTTP/2 error code (RFC-7540 section 7) sent by peer.
  *      `enum aws_http2_error_code` lists official codes.
+ * @param debug_data The debug data sent by peer. It can be empty. (Note: there is no guarantee about the lifetime of it
+ *      after the callback invoked. To keep it safe for further reference, caller should keep a deep copy)
```

Comment:
just suggesting more concrete language:
~~there is no guarantee about the lifetime of it after the callback invoked~~
(NOTE: this data is only valid for the lifetime of the callback. Make a deep copy if you wish to keep it longer.)

Reply:
oh! also, henso suggested to me a few months back changing my writing style to avoid "passive voice" in documentation. Basically, you write directly to the user, telling them "You should do X" instead of "The user should do X".

I used to think it looked informal, but now I like it. I'm noticing that docs written that way are easy to follow, as a user. And as a writer I find it nice and simple to just tell someone what to do, instead of writing about the thing they need to do in a weird 3rd person way.